src/govuk/components/task-list/template.njk

```diff
@@ -0,0 +1,28 @@
+{% set idPrefix = params.idPrefix if params.idPrefix else "task-list" %}
+<ul class="govuk-task-list">
+  {% for item in params.items %}
+    {% if item.hint %}
+      {%- set hintId = idPrefix + "-" + loop.index + "-hint" -%}
+    {% endif %}
+    {%- set statusId = idPrefix + "-" + loop.index + "-status" -%}
+    <li class="govuk-task-list__item {%- if item.href %} govuk-task-list__item--with-link{% endif %}">
+      <div class="govuk-task-list__task-name-and-hint">
+        {% if item.href %}
+          <a class="govuk-link govuk-task-list__link" href="{{ item.href }}" aria-describedby="{{ hintId + " " if item.hint }}{{ statusId }}">{{ item.title.html | safe if item.title.html else item.title.text }}</a>
+        {% else %}
+          <div class="govuk-task-list__task-no-link">{{ item.title.html | safe if item.title.html else item.title.text }}</div>
+        {% endif %}
+        {% if item.hint %}
+          <div id="{{ hintId }}" class="govuk-task-list__task_hint">
+            {{ item.hint.html | safe if item.hint.html else item.hint.text }}
+          </div>
+        {% endif %}
+      </div>
+      <div class="govuk-task-list__status-container">
+        <span id="{{ statusId }}" class="govuk-task-list__status {{ item.status.classes }}">
```

Comment:
A quick question about the use of `span` here that was prompted by the [tag redesign](https://github.com/alphagov/govuk-frontend/pull/3502). The tag currently uses a `strong` as its containing element. The history around why this was chosen is very wooly as it was pulled from the legacy frameworks but the tea are confident that this is for progressive enhancement reasons to maintain some form of visual delegation when css isn't available ie: if you turn css off, the tag will still be bold. If you were to use the redesigned tag component, this would turn into a `strong`.

My assumption here is that a `span` was chosen here just because it semantically makes sense and swapping it out for a `strong` won't disrupt the implementation (they're both inline elements so it doesn't make a difference) but I wanted to prompt on that to make sure this wasn't going to be disruptive. FWIW my personal take is that plonking the tag in as a `strong` is perfectly fine and we don't need to make any destructive changes to the markup of the tag component at present.

Reply:
@owenatgov hi - yep I swapped it to a span just because it no longer visually appeared bold.

However with our decision to also update the Tag component, I'm going to update the Task list component to actually use the Tag component for the statuses (as a sub-component), so it'll swap to using a `<strong>` element. I'd agree with your take that that's fine, and possibly even preferable where the CSS isn't available.